src/js/api/api.js

```diff
@@ -967,6 +967,15 @@ Object.assign(Api.prototype, /** @lends Api.prototype */ {
     },
 
     /**
+     * Extracts the video tag in the foreground.
+     * @returns {object|null} videoTag - the HTML <video> element in the foreground.
```

Comment:
It's an `Element`, not object

https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector